src/main/java/org/codeforamerica/messaging/models/Message.java

```diff
@@ -0,0 +1,40 @@
+package org.codeforamerica.messaging.models;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.*;
+import org.codeforamerica.messaging.utils.RegexPatternStrings;
+import org.codeforamerica.messaging.validators.ValidMessage;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import lombok.Builder;
+import java.time.OffsetDateTime;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+@ValidMessage
+public class Message {
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+    @Pattern(regexp = RegexPatternStrings.PHONE_NUMBER_REGEX)
+    String toPhone;
+    @Email
+    String toEmail;
```

Comment:
If `subject` is present and `toEmail` is not, should we reject it? Or silently not send an email?

Reply:
Right now, it should if `subject` is present and `toEmail` is not:
1. It would ignore it silently if `toPhone` is present.
2. It would reject it if `toPhone` is absent.

I am open to changing this.